### ISSUE TITLE
Fail to plot analysis for only one chr 

### ISSUE THREAD

**Issue by mpierrejean** [broadinstitute/ichorCNA#57]
It seems that the plotting function doest not work with only one chromosome is provided.
[Line 348](https://github.com/broadinstitute/ichorCNA/blob/master/R/plotting.R#L348) in plotting function.

Could you test if the length of the chromosome list is equal to one?

Thanks

**Comment by gavinha** [comment]
Hi @mpierrejean 

For plotting one chromosome, it shouldn't be invoking `getGenomeWidePositions()`. Can you please let me know how you're trying to plot one chromosome and which functions you are using to do this?  This will help me figure out why you're running to this issue.

Thanks,
Gavin

**Comment by mpierrejean** [comment]
Hi @gavinha 

I have the following error:

> Error in chrBkpt[i] <- prevChrPos : replacement has length zero
> Calls: plotGWSolution -> plotCNlogRByChr -> getGenomeWidePositions

We performed tests on short bam files to create a pipeline with ichorCNA. The bam files only contain the chr20. 

```shell
ichorCNA.sh \
    -f "${TEST_DIR}/conf.txt" \
    -ctrl "${TEST_DIR}/control.bam" \
    -o "${TEST_DIR}/result" \
    -chr 'chr20'
```
In conf.txt there is one line `CRPC1=tests/test1.bam,tests/test2.bam`

ichorCNA.sh calls the function `runIchorCNA.R` in `ichorCNA/scripts/ichorCNA/scripts/runIchorCNA.R`.

I hope I gave enough details. 
I will not available for the next 4 weeks. 

Thanks.
